exercises/elm-1-b/README.md

```diff
@@ -0,0 +1,35 @@
+## Introduction
+
+This is Part 1 of our research into how people write Elm differently. This exercise should take 10-15 minutes. There is no right or wrong way to approach it - just do what feels the most natural to you. Feel free to use books, Google or Stack Overflow, just like you would if you were programming normally, but please don't actively look for a solution to the problem as this will negatively affect the research.
+
+## Instructions
+
+You are the proud owner of an amazing pizzeria but your website is missing the menu.
```

Comment:
Love this background story!

Reply:
Hi Erik, I've already taken a look at this, and have just approved it. Not that I think matthieu really needs me to check his work :)

Reply:
we always need another pair of eyes ;)